book/src/framework/components/network_isolation.md

```diff
@@ -0,0 +1,13 @@
+# Containers Network Isolation
+
+Some components can be isolated from internet. Since some of them doesn't have `iptables` and must be accessible from local host for debugging we isolate network on DNS level
+
+```
+[jd]
+  # JobDistributor is isolated by default, this flag MUST not be changed!
+  no_dns = true
+
+[[nodesets]]
+  # NodeSet DNS can be isolated if needed
+  no_dns = true
+```
```

Comment:
any chance we might want/need it also for pg databases?

Reply:
I don't think so, we only need to prevent CL nodes from applying changed manifests to CL nodes.